src/Npgsql/NpgsqlConnectionStringBuilder.cs

```diff
@@ -448,7 +449,7 @@ public SslMode SslMode
                 SetValue(nameof(SslMode), value);
             }
         }
-        SslMode _sslMode;
+        SslMode _sslMode = SslMode.Prefer;
```

Comment:
IIRC this doesn't need to be done (we already initialize everything based on [DefaultValue], see all other properties)